process-engine/bpmn-studio
# 🐛Fix Token Viewer Update

**Changes:**

1. Only update token viewer when the element can have a token history.
2. Fix overlay click handler in LET.

PR: #PullRequest

## How can others test the changes?

1. Open a Diagram.
2. Start it. (It should have a UserTask in it)
2.1. See that there are less faulty request in the PE log.
2.2 Notice that you can click on the UserTask (not on the play button) to update the Token Viewer without opening the Dynamic Ui Modal.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
